lib/rules/comment-empty-line-before/index.js

```diff
@@ -61,7 +60,7 @@ const rule = (primary, secondaryOptions, context) => {
 
 			// Optionally ignore stylelint commands
 			if (
-				comment.text.startsWith(stylelintCommandPrefix) &&
+				comment.text.startsWith(context.configurationComment || DEFAULT_CONFIGURATION_COMMENT) &&
```

Comment:
Can we use the `isConfigurationComment` util here instead?

Reply:
Fixed